.gitignore

```diff
@@ -0,0 +1,393 @@
+﻿## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
+## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
+
+# User-specific files
```

Comment:
Why so many entry in gitignore ?

Reply:
This is a .gitignore template for vs projects thats excludes all environment-related files and build artifacts.

source: https://github.com/github/gitignore/blob/master/VisualStudio.gitignore